tests/option.rs

```diff
@@ -102,14 +102,14 @@ fn option_test_normal_option() {
 fn option_test_defer_option() {
     #[derive(Copy, Clone, Default, Miniconf)]
     struct S {
-        #[miniconf(defer)]
+        #[miniconf(defer(0))]
```

Comment:
Isn't defer(0) the same as not putting an attribute at all? Or is this only different for `Option`?

Reply:
Defer(0) calls the miniconf API.
No miniconf attribute calls serde directly.
For option, the former is the "mask single serde item" use case.